apps/src/blockly/addons/cdoFieldLabel.ts

```diff
@@ -8,15 +8,27 @@ import * as GoogleBlockly from 'blockly/core';
 // Artist, and Play Lab. We can potentially add additional handling of
 // this argument to this class in the future should we need it.
 export default class CdoFieldLabel extends GoogleBlockly.FieldLabel {
+  // An override for legacy labs like Jigsaw where the rendered block size is constant.
```

Comment:
Do any other legacy labs actually use this? If not, is it more accurate to make this comment `An override for legacy labs (Jigsaw as of 11/24) where the rendered block size is constant` or something?

Reply:
Yes - It's actually used in Maze/Artist too, for cardinal directions (N/E/S/W), but it doesn't seem to have much impact on the rendered block because the label text value is always just one character.
It's also used in some CS in Alg blocks that I just found still kicking around.

Reply:
If you're the only person capable of filling in the "legacy labs like Jigsaw" list, maybe worth listing what you know even if it's verbose, but no strong feelings 😄

Reply:
Yeah, I mean if you search the repo for `fixedSize` there's only four instances, so I think it's fairly easy to figure out where it's being used.